tests/components/test_wake_on_lan.py

```diff
@@ -0,0 +1,48 @@
+"""Tests for Wake On LAN component."""
+import asyncio
+from unittest import mock
+
+import pytest
+
+from homeassistant.setup import async_setup_component
+from homeassistant.components.wake_on_lan import (
+    DOMAIN, SERVICE_SEND_MAGIC_PACKET)
+
+
+@pytest.fixture
+def mock_wakeonlan():
+    """Mock mock_wakeonlan."""
+    module = mock.MagicMock()
+    with mock.patch.dict('sys.modules', {
+        'wakeonlan': module,
+    }):
+        yield module
+
+
+@asyncio.coroutine
+def test_send_magic_packet(hass, caplog, mock_wakeonlan):
+    """Test of send magic packet service call."""
+    mac = "aa:bb:cc:dd:ee:ff"
+    bc_ip = "192.168.255.255"
+
+    yield from async_setup_component(hass, DOMAIN, {})
+
+    yield from hass.async_add_job(
+        hass.services.call, DOMAIN, SERVICE_SEND_MAGIC_PACKET,
+        {"mac": mac, "broadcast_address": bc_ip})
```

Comment:
So I have to call `hass.async_block_till_done`after each call, isn't it? I'll change that